.github/workflows/build-and-publish-bootstrap-container.yml

```
@@ -0,0 +1,52 @@
+name: Build and publish EESSI Docker image for bootstrapping Gentoo Prefix
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - Dockerfile.bootstrap-prefix-centos8
+      - bootstrap-prefix.sh
+
+  pull_request:
+    branches:
+      - main
+    paths:
+      - Dockerfile.bootstrap-prefix-centos8
+      - bootstrap-prefix.sh
+ 
+jobs:
+  build_and_publish:
+    name: Build and publish image
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+      contents: read
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to GitHub Container Registry
+        if: github.event_name != 'pull_request'
+        uses: docker/login-action@v1 
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Convert and store repository owner in lowercase
+        run: |
+          echo REPOSITORY_OWNER=$(echo ${{ github.repository_owner }} | tr '[:upper:]' '[:lower:]') >> $GITHUB_ENV
```

I would use double quotes here, which makes this a bit easier to parse (for humans):

```suggestion
          echo "REPOSITORY_OWNER=$(echo ${{ github.repository_owner }} | tr '[:upper:]' '[:lower:]')" >> $GITHUB_ENV